doc/whatsnew/v1.0.0.rst

```diff
@@ -7,43 +7,75 @@ changes.
 Changelog
 ---------
 
-Enhancements
-............
+Major changes
+.............
 
-* fully python3 compatible -- one code-base for both python 2 and python 3
-* package name and the import name match -- now use ``import pydicom`` rather
-  than ``import dicom``
-* ``read_file`` changed to ``dcmread`` for greater clarity.  Similarly ``write_file`` becomes ``dcmwrite``.  Previous names still available for backwards compatibility
+* full Python 3 compatibility - one code-base for both Python 2 and Python 3
+* package name and the import name now match -- use ``import pydicom``
+  rather than ``import dicom``
 * optional GDCM support for reading files with compressed pixel data
-* optional Pillow, jpeg_ls support for reading some compressed pixel data files
-* cleaned up dicom dictionary code, old non-dicom-keyword code removed
-* dicom dictionary updated to 2017c
+  (:issue:`18`)
+* optional Pillow and jpeg_ls support for reading some compressed pixel data
+  files
+* support for both decompressing a dataset inline, and return a decompressed
+  dataset
```

Comment:
slight error in the language here, I think -- we don't return a decompressed dataset, but convert the dataset to decompressed in-place, no returned values.  The other option is for pixel_array to return decompressed pixel data without modifying the dataset otherwise.

Reply:
Also, I'd like to see a link here (right where the new handling of compressed pixel data is announced) to a page with details/warnings - including @rhaxton's list in #542 of what works with each kind of compression, and what is giving uncertain results.  With that link, I'd suggest we strongly advise that people read that page before using these handlers.

Reply:
> slight error in the language here

Ah yes, of course. I will just remove the part about returning the compressed data, that is covered by the prior 2 items.
> Also, I'd like to see a link here

Good idea - I will add something in the evening.